packages/apps/src/Apps.tsx

```diff
@@ -34,11 +34,11 @@ const Wrapper = styled.div`
   min-height: 100vh;
 
   header {
-    margin-bottom: 0.8rem;
+    margin-bottom: 1.4rem;
     text-align: center;
 
     ${media.TABLET`
-      margin-bottom: 1.1rem;
+      margin-bottom: 2.8rem;
```

Comment:
This "feels" a bit too much, `2.25rem` or even, `2rem` "feels" much bit better. (Specifically in the case where we have no summary on top, the spacing is too much - 2 seems to give a happy medium)

Apart from that, lgtm

Reply:
i've pushed a commit changing it to 2rem